crates/solidity/testing/snapshots/cst_output/Block/unchecked/generated/0.4.11-failure.yml

```diff
@@ -5,7 +5,7 @@ Source: >
 
 Errors: # 1 total
   - >
-    Error: Expected Semicolon.
+    Error: Expected Ampersand or AmpersandAmpersand or AmpersandEqual or Asterisk or AsteriskAsterisk or AsteriskEqual or BangEqual or Bar or BarBar or BarEqual or Caret or CaretEqual or Equal or EqualEqual or GreaterThan or GreaterThanEqual or GreaterThanGreaterThan or GreaterThanGreaterThanEqual or GreaterThanGreaterThanGreaterThan or GreaterThanGreaterThanGreaterThanEqual or LessThan or LessThanEqual or LessThanLessThan or LessThanLessThanEqual or Minus or MinusEqual or Percent or PercentEqual or Plus or PlusEqual or Semicolon or Slash or SlashEqual.
```

Comment:
`unchecked` can only accept a block afterwards. Is the new scanner correct in attempting these?

Reply:
It thinks `unchecked` in this version is an identifier, which is actually the correct thing to do. This work fixes a number of bugs that we didn't realise we had.